tools/build_utils/fypp

```diff
@@ -37,7 +37,7 @@ Python, one usually interacts with the following two classes:
 * `Fypp`_: The actual Fypp preprocessor. It returns for a given input
   the preprocessed output.
 
-* `FyppOptions`_: Contains customizable settings controling the behaviour of
+* `FyppOptions`_: Contains customizable settings controlling the behaviour of
```

Comment:
please send fixes for Fypp directly to https://github.com/aradi/fypp

Reply:
aradi/fypp#6